packages/@n8n_io/eslint-config/base.js

```diff
@@ -419,19 +419,9 @@ const config = (module.exports = {
 		 */
 		'prefer-spread': 'error',
 
-		/**
-		 * https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/docs/rules/no-unused-vars.md
-		 */
+		// These are tuned off since we use `noUnusedLocals` and `noUnusedParameters` now
 		'no-unused-vars': 'off',
-		'@typescript-eslint/no-unused-vars': [
-			process.env.CI_LINT_MASTER ? 'warn' : 'error',
-			{
-				argsIgnorePattern: '^_',
-				destructuredArrayIgnorePattern: '^_',
-				varsIgnorePattern: '^_',
-				ignoreRestSiblings: true,
-			},
```

Comment:
My only doubt was whether we could exempt unused params with underscore, but it looks like the TS setting supports this out of the box 👍🏻